tool/planet/etcd.go

```diff
@@ -751,12 +752,38 @@ func systemctlCmd(ctx context.Context, operation, service string, args ...string
 		"service":   service,
 	}).Info("Execute systemctl.")
 	if err != nil {
-		return trace.Wrap(err, "failed to execute systemctl: %s", out).AddFields(map[string]interface{}{
+		return "", trace.Wrap(err, "failed to execute systemctl: %s", out).AddFields(map[string]interface{}{
```

Comment:
There's currently a single caller that uses this return yet you had to change every other place where the API is used.
How about adding a `systemctlCmdWithOutput` that returns the output and use it where the output is required:
```go
func systemctlCmd(ctx context.Context, operation, service string, args ...string) error {
	_, err := systemctlCmdWithOutput(ctx, operation, service, args...)
	if err != nil {
		return trace.Wrap(err)
	}
	return nil
}

func systemctlCmdWithOutput(ctx context.Context, operation, service string, args ...string) (output string, err error) {
	args = append([]string{operation, service}, args...)
	out, err := exec.CommandContext(ctx, "/bin/systemctl", args...).CombinedOutput()
	log.WithFields(log.Fields{
		"operation": operation,
		"output":    string(out),
		"service":   service,
	}).Info("Execute systemctl.")
	if err != nil {
		return "", trace.Wrap(err, "failed to execute systemctl: %s", out).AddFields(map[string]interface{}{
			"operation": operation,
			"service":   service,
		})
	}
	return string(out), nil
}
```

Reply:
I just wanted to avoid duplication of code and didn't think about that implementation. Thanks for the suggestion!